MailCore/Utils/UTType+Extension.swift

```diff
@@ -0,0 +1,30 @@
+/*
+ Infomaniak Mail - iOS App
+ Copyright (C) 2022 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Foundation
+import UniformTypeIdentifiers
+
+public extension UTType {
```

Comment:
There will be soon an UTI in core that can be extended and used by platform apps. (after share ext is meged)

Reply:
discussed with @valentinperignon , no need for ikMail. Will check later if we need it for kDrive.